src/server/pps/cmds/cmds.go

```diff
@@ -438,6 +439,27 @@ each datum.`,
 		follow      bool
 		tail        int64
 	)
+
+	// prettyLogsPrinter helps to print the logs recieved in different colours
+	prettyLogsPrinter := func(message string) {
+		informationArray := strings.Split(message, " ")
+		if len(informationArray) > 1 {
+			debugLevel := strings.ToLower(informationArray[1])
+			if debugLevel == "info" {
+				color.New(color.FgGreen).Println(message)
+			} else if debugLevel == "warning" {
+				color.New(color.FgYellow).Println(message)
+			} else if debugLevel == "error" {
+				color.New(color.FgRed).Println(message)
+			} else {
+				fmt.Println(message)
+			}
+		} else {
+			fmt.Println(message)
+		}
+
```

Comment:
You'll still want to unset the color after each line, otherwise the colors will persist beyond the printed lines

Reply:
Here I am not setting the color rather I am just making the object using the function and printing my message and then letting the object go. I tried something similar.

```
color.New(color.FgRed).Println("message")
fmt.Println("hello world")
```

This works alright, am I missing something? :sweat_smile:

Reply:
Hahaha cool 😊